.github/workflows/build-test.yml

```
@@ -20,7 +20,9 @@ jobs:
       - name: Setup .NET Core
         uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: '8.0.x'
+          dotnet-version: |
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 23: found a tab character where an indentation space is expected

(syntax-check)